src/Workspaces/CSharp/Portable/Classification/EmbeddedLanguages/CSharpFallbackEmbeddedLanguageClassifier.cs

```diff
@@ -2,20 +2,16 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Composition;
 using Microsoft.CodeAnalysis.Classification;
 using Microsoft.CodeAnalysis.CSharp.EmbeddedLanguages.LanguageServices;
-using Microsoft.CodeAnalysis.Host.Mef;
 
 namespace Microsoft.CodeAnalysis.CSharp.Classification
 {
-    [ExportEmbeddedLanguageClassifier(PredefinedEmbeddedLanguageClassifierNames.Fallback, LanguageNames.CSharp), Shared]
```

Comment:
instead of mef importing the fallback classifiers.  they are just built in as we know we always need them for c#/vb.